### ISSUE TITLE
Alteration of the gut fecal microbiome in children living with HIV on antiretroviral therapy in Yaounde, Cameroon

### ISSUE THREAD

**Issue by Scholarpat** [waldronlab/BugSigDBcuration#447]
Alteration of the gut fecal microbiome in children living with HIV on antiretroviral therapy in Yaounde, Cameroon - William Baiye Abange – Scientific Reports

https://www.nature.com/articles/s41598-021-87368-8

**Comment by ToluwalaseAO** [comment]
@SvetlanaUP , can I get this issue assigned to me?

**Comment by ToluwalaseAO** [comment]
Hi @AishatBolarinwa, thank you for reaching out. I'll be sure to get in touch if I encounter any challenges.

**Comment by ToluwalaseAO** [comment]
Hi. @SvetlanaUP my second contribution is ready for review. Thank you.

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_1179 ready for review.

**Comment by SvetlanaUP** [comment]
Hi @ToluwalaseAO, could you please confirm that only Experiment 1 with no signatures is recorded for this curation. Thank you.

**Comment by ToluwalaseAO** [comment]
Hi @SvetlanaUP,No I added Experiment 2  with signatures which is also recorded for this curation..

**Comment by SvetlanaUP** [comment]
@ToluwalaseAO there is no https://bugsigdb.org/Study_1179/Experiment_2

**Comment by ToluwalaseAO** [comment]
@SvetlanaUP It looks like it wasn’t saved. I will go ahead with experiment 1..

**Comment by AleruDivine** [comment]
Hi @ToluwalaseAO, I'd love to collaborate with you on completing this curation if that's okay with you.

**Comment by AleruDivine** [comment]
Thank you @SvetlanaUP I'll reach out to @ToluwalaseAO on Slack

**Comment by AleruDivine** [comment]
Good afternoon @ToluwalaseAO 
I’ve been trying to get a hold of you on slack for a while now regarding this curation but you haven’t responded so maybe you’ll get an email notification. I hope everything is okay. 🙏

**Comment by AleruDivine** [comment]
Alright @SvetlanaUP finalize this curation. 

Thank you so much!

**Comment by AleruDivine** [comment]
Hello @SvetlanaUP I believe this study is ready for review.
Link: https://bugsigdb.org/Study_1179

Thank you so much!

**Comment by KateRasheed** [comment]
Good morning @AleruDivine . Great job on this curation. Just few things to correct on this study:

For experiment 1 and 2, please state the full name of HIV, NNRTI, PI/r.

For signature 1 in experiment 1, Ruminococcaceae, lachnospiraceae and Clostridiaceae were increased.

For signature 2 in experiment 1, Clostridiaceae was not decreased.

Thank you so much.

**Comment by AleruDivine** [comment]
Thank you so much for the review @KateRasheed.
The full names for HIV, NNRTI, and PI/r have been added and Clostridiaceae added to signature 1 (increased).

I'd like for you to check Ruminococcaceae, Lachnospiraceae again because they are more on the negative side (less than 0) which means decreased for HIV positive.

Also see this from the discussion "Similarly, we observed broad decreases in Prevotellaceae and Lachnospiraceae relative abundance in our HIV-infected cohort"

**Comment by KateRasheed** [comment]
Yes I see that they are more on the decreased side but they are also in the increased side. This is also a case of taxa seen in the increased and decreased.

This was why I thought it be added to increased as well since it was observed in the positive. @AleruDivine

**Comment by AleruDivine** [comment]
@KateRasheed Yes, they are present in both HIV-uninfected and HIV-infected groups, but more decreased for HIV-infected group.  
They are more on the decreased side just like you mentioned.

**Comment by KateRasheed** [comment]
Thank you @AleruDivine . This review is complete @SvetlanaUP

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_1179 reviewed.

**Comment by SvetlanaUP** [comment]
@AleruDivine could you please specify here the updates that you just made.

**Comment by AleruDivine** [comment]
My apologies @SvetlanaUP
In signatures 1 and 2 of Experiment 1, I updated the increased and decreased signatures to include only _Akkermansia muciniphila_ and _Faecalibacterium prausnitzii_ based on Chloe's explanation during office hours. 

The ASVs are represented as circles (at the genus or species level) and were color-coded according to the label provided next to the figure. So, I curated A. muciniphila (class Verrucomicrobiae)to indicate an increase and F. prausnitzii (class Clostridia) to indicate a decrease, as shown in Figure 4A.
